jujugui/static/gui/src/app/app.js

```diff
@@ -1347,7 +1345,7 @@ YUI.add('juju-gui', function(Y) {
             envResolved={this.env.resolved.bind(this.env)}
             serviceRelations={
               relationUtils.getRelationDataForService(this.db, service)}
-            addGhostAndEcsUnits={utils.addGhostAndEcsUnits.bind(
+            addGhostAnlocalTypedEcsUnits={utils.addGhostAndEcsUnits.bind(
```

Comment:
Did you mean to update this? I couldn't see any changes to the component prop name.